### ISSUE TITLE
Update `integration tests` for `GitHub API` calls

### ISSUE THREAD

**Issue by r002** [studydash/cards#118]
## Thursday
Continuing the testing uplift effort, today I [updated](https://github.com/r002/storyline-api/blob/2d7a9d4f90a4373f20411bbe4cc7180047ce0b16/ghservices/dataservice_test.go#L83-L139) the integration tests for all of my code that interacts with the `GitHub API` to make them more robust.

### Tests:
- Create a card
  - Verify that the card is labeled with the correct `day` and milestoned as `Daily Accomplishment`
  - ~~Verify that the card has propagated to `Firestore`~~
- **Individual Tests:**  
  - `$>` `go test .\ghservices\ -run CreateCard -v -count=1`
  - `$>` `go test .\ghservices\ -run UpdateCard -v -count=1`

### Changes:
Instead of constantly overwriting the `ghUpdates/lastestUpdate` doc, begin creating new documents for each update.  Sort by `dt` and have the `dashboard` GUI only render the most recent `lastestUpdate` doc.

<br />

## References 🔗
- https://docs.github.com/en/rest/reference/issues#list-repository-issues
- **Subtests:** https://courses.calhoun.io/lessons/les_twg_les_20
- https://www.geeksforgeeks.org/how-to-get-current-weekday-in-golang/
- https://play.golang.org/p/eHzirKl1oIx

<br />

## Extra/Fun 🎈🎉
Shaza and I got to the third season of _The Kominsky Method_ a few days ago but there was a major personnel change in the casting!  I won't spoil anything but the character was one of our favorites.  I wanted to keep watching (S3 is the final season) but Shaza didn't want to continue so we switched over to a new show, [_Grace and Frankie_](https://www.youtube.com/watch?v=CDv6PRi1SgQ).  It's excellent too!  Just finished S1E3 earlier tonight! 🥳

![image](https://user-images.githubusercontent.com/45280066/122514740-6f2d6c00-cfda-11eb-8056-77e3c3af541a.png)

**Comment by anitabe404** [comment]
Woohoo. Writing tests means our group is officially legit. lol. Testing is where most of the magic/fun happens. 

Side note: Did you hear about all the HBO Max customers that got an integration test email by accident?

**Comment by r002** [comment]
Haha, yep!  Testing is where all the fun's at!  😄

Also-- what's bizarre is I'm not even an HBO Max Customer!! 🤷‍♂️

![image](https://user-images.githubusercontent.com/45280066/122507703-a72eb200-cfce-11eb-814c-1c3e7ca55fd9.png)

**Comment by anitabe404** [comment]
@r002 Okay, now that makes it super weird. 😂

Earlier this week, I learned about clear gifs and now I'm paranoid about opening emails (and websites). 😭. Is someone running a large-scale scam over at HBO?

**Comment by r002** [comment]
Yeah-- clear gifs, tracking pixels, and [web beacons](https://en.wikipedia.org/wiki/Web_beacon) in general are pretty underhanded.  Thankfully, with adblocking tools, most of that can be stopped.  Just earlier this week, I was reading [a post by DHH](https://world.hey.com/dhh/email-spy-pixels-are-dead-now-that-apple-will-follow-hey-eb66b759) talking about how they inspired Apple to kill all spy pixels in the next Apple Mail version on MacOS that'll be releasing soon.  Better late than never!

Re the HBO Max snafu, thankfully, it looks like it wasn't a phishing scam and was only [the intern!](https://www.msn.com/en-us/tv/news/hbo-max-subscribers-confused-and-amused-by-mysterious-integration-test-email-supernumber-1/ar-AALayCS)  😂

![image](https://user-images.githubusercontent.com/45280066/122510449-a0566e00-cfd3-11eb-9ff8-f59e371937a4.png)
